src/components/Footer/HeroFooter.jsx

```diff
@@ -1,22 +1,76 @@
-import React from 'react'
-import styles from './Footer.module.css'
-import footerImage from '../../assets/bg-pattern-home-6-about-5.svg'
-import Link from 'react'
+import React from "react";
+import { useEffect, useState } from "react";
+import Typography from "../utility/Typography/Typography";
+import styles from "../Footer/HeroFooter.module.css";
+import logo from "../../assets/logo.svg";
+import facebook from "../../assets/icon-facebook.svg";
+import pinterest from "../../assets/icon-pinterest.svg";
+import twitter from "../../assets/icon-twitter.svg";
+import toggleButton from "../../assets/on-button-removebg-preview.png";
+import { Link } from "react-router-dom";
 
 function HeroFooter() {
-    return (
-        <div className={styles.footerContent}>
-       <img src={footerImage} alt="" className="footerImg" />
-            <div className={styles.footerText}>
-                <h2>Ready to get started?</h2>
-
-                <div className={styles.btn}>
-                    <button>contact us</button>
-                </div>
-            </div>
-            
+  const [theme, setTheme] = useState(
```

Comment:
Its a very nice implementation of the dark mode.  Unfortunately we don't have the design for the light mode for the entire project. So we cant keep the dark mode in scope for the footer

Reply:
Sure I will make the necessary changes as stated